packages/cdk/webworker/src/worker.decorator.ts

```diff
@@ -0,0 +1,15 @@
+import { Any, Descriptor } from '@angular-ru/cdk/typings';
+
+import { WebWorkerThreadService as Thread } from './worker-thread.service';
+
+export function WebWorker() {
```

Comment:
don't see, have you tested this anywhere?

Reply:
I will add tests in next commit